### ISSUE TITLE
Deleting objects in design tab, moves the actual screen I am working to the first screen

### ISSUE THREAD

**Issue by monoescobar** [thunkable/thunkable-issues#715]
Thanks so much for filing an issue or feature request! Please fill out the following (wherever relevant):

The more info, the faster we will be able to address it!

### Platform
x.thunkable.com

### Issue/Request Origin
Where did you get this issue? Email, Intercom, or Community?

### Steps to Reproduce
1. I am in let say in the screen working in the design tab.
2. I delete a component, for example a Colum or row or label.
3. right after the element is deleted, autiomaticaly the selction of the screen ai am workig move to the first screen, so I have to go back ti the thrid screen, or the one I was working on.

### Project Link
Any project link yo cna see this behaviour. 

### Expected Behavior
stay in the same screen in the below or abobe element of the one element that was deleted. 

### Actual Behavior
moves to the first screen when any element in any other screen is dleted in the design tab
```
  STACK TRACE AND/OR SCREENSHOTS
```
https://www.loom.com/share/fc9d8d6726c74717a22a5d90068294a9

**Comment by Jemug** [comment]
I have the same issue, it's very annoying

**Comment by StevenPurn** [comment]
Thanks for letting us know! We believe we have identified the issue and are working on a fix now. Sorry about that!

**Comment by tatiang** [comment]
Hoping this gets fixed soon! It's unsettling to delete a component and then have the screens switch automatically.

**Comment by AmigoDeluxe** [comment]
Yeah. I noticed this as well. Bummer.

**Comment by tatiang** [comment]
It finally happened. I accidentally deleted a component I needed. 

I first intentionally deleted a component from screen #2 and then accidentally deleted a different component from screen #1 because the screens auto-switched and then looked similar enough to each other.

This is frustrating and I hope it gets fixed soon.

**Comment by tatiang** [comment]
Bumping this because it's still a problem. Now that I'm developing an app with many screens, it's even more of an issue that whenever I delete a component, I'm taken to the first screen and have to navigate back to wherever I was and try to remember what I was in the middle of doing.

**Comment by StevenPurn** [comment]
Sorry about that! I thought another member of our team had worked on and resolved this issue previously. I'm not sure what happened but it seems like the fix didn't get in. I'll make it a priority to get this fixed this week. Though important to note, that doesn't mean it will get released this week. I'll post back here once I have an update.

**Comment by tatiang** [comment]
Thanks for the update, @StevenPurn! I'm glad it's getting looked at.

**Comment by StevenPurn** [comment]
Just wanted to follow up on this. We have a fix for this ready to go. I think we have a release going out today so I'm not sure if it will make it in time to go out today but, if not, it should be included in the next release we put out. Sorry it ended up taking so long! Thanks for making sure we got to it eventually.

I'm going to leave this open until we've confirmed the fix is released and working as expected.
